data/monthly.csv

```diff
@@ -1,99 +1,734 @@
 Date,Total Equity,Domestic Equity,World Equity,Hybrid,Total Bond,Taxable Bond,Municipal Bond,Total
+2007-01-31,27441,5976,21464,5498,15047,12227,2820,47986
```

Comment:
Why we have 4 rows per month with identical date, but different values, see previous version which has 1 row per month, could you check your script?

Reply:
@Mikanebu I have updated the data